readme.md

```diff
@@ -273,9 +273,9 @@ yarn build-all
 ```
 
 #### installing protobuf
-If changing audit output, you'll need to have v3.7.1 of the protocol-buffer/protobuf compiler installed. (v3.7.1 is known to be compatible, and 3.11.x is known to be **not** compatible.). 
+If changing audit output, you'll need to have v3.7.1 of the protocol-buffer/protobuf compiler installed. (v3.7.1 is known to be compatible, and 3.11.x is known to be **not** compatible.).
 
-Homebrew should be able to install it correctly: `brew install protobuf@3.7.1`
+Homebrew should be able to install it correctly: `brew install protobuf@3.7`
```

Comment:
were these changes on purpose?

Reply:
yes drive by fix as I needed to run this on my new machine and `brew install protobuf@3.7.1` doesn't work, it only supports @3.7

Reply:
> yes drive by fix as I needed to run this on my new machine and `brew install protobuf@3.7.1` doesn't work, it only supports @3.7

Looks like `@3.7` is [the official way to refer to the older version now](https://formulae.brew.sh/formula/protobuf)

(our tests not working as desired on the lastest protobuf release it going to come around to bite us sooner or later...:)